### PR TITLE
Add NEWS file for 3.0 changes

### PR DESCRIPTION
**Pull request by ehiggs**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# 3.0.0
+    * Command line reoganization. `hod` now uses a command/subcommand style
```
Comment (boegel) [inline NEWS.md]: typo: `reoganization` => `reorganization`

File: NEWS.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      interface which reduces the number of possible conflicting flags.
+      `hod create`
```
Comment (boegel) [inline NEWS.md]: what is this dangling here?

also, point to hod.readthedocs.org/en/latest/Command_line_interface.html

File: NEWS.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    * Batch mode so users can start a cluster, run a job, and quit.
```
Comment (boegel) [inline NEWS.md]: mention `hod batch`

File: NEWS.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    * `hod connect` so users can connect and reconnect without the risk of
+      losing their screen session.
+    * Job labels. Used in conjunction with `hod connect`. Users can also relabel
+      their jobs using `hod label`
```
Comment (boegel) [inline NEWS.md]: `hod relabel`

File: NEWS.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+    * Support for IPython notebooks with Spark as a back end.
+    * Support for HBase.
+    * [Extensive documentation](http://hod.readthedocs.org) covering Windows, OS
+      X, and Linux for using `hod` web services.
```
Comment (boegel) [inline NEWS.md]: point to http://hod.readthedocs.org/en/latest/Connecting_to_web_UIs.html directly?

File: NEWS.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+    * Internal improvements that might not be visible to users:
+      * Support for zipped eggs. This speeds up the command line experience when
+        running `hod` from a shared file system since there are less files to poke
+        through.
+      * Support for Python 2.6.
+      * Using Jenkins for continuous integration for improved quality..
```
Comment (boegel) [inline NEWS.md]: double `.`

**Comment by boegel** [comment]
@ehiggs: please make a separate PR with the changes to `setup.py` and `tox.ini`, and include a clear comment in the files themselves why we're locking things to specific versions of `vsc-base` and `mpi4py`

**Comment by ehiggs** [comment]
I split out the NEWS and the build fix PRs.

**Pull request merged**